### PR TITLE
Add backup extension support for provider openstack

### PR DESCRIPTION
**Pull request by swapnilgm**
Signed-off-by: Swapnil Mhamane <swapnil.mhamane@sap.com>

**What this PR does / why we need it**:
This PR adds the backup extension support for provider Openstack swift
**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:
Please refer to #123.

**Release note**:

```improvement operator
NONE
```

### CODE REVIEW

File: controllers/provider-openstack/charts/provider-openstack/templates/rbac.yaml [hunk 1]
```diff
@@ -9,11 +9,15 @@ rules:
 - apiGroups:
   - extensions.gardener.cloud
   resources:
+  - backupbucket
```
Comment (rfranzke) [inline controllers/provider-openstack/charts/provider-openstack/templates/rbac.yaml]:
```suggestion
  - backupbuckets
```

File: controllers/provider-openstack/charts/provider-openstack/templates/rbac.yaml [hunk 2]
```diff
@@ -9,11 +9,15 @@ rules:
...
+  - backupbucket/status
```
Comment (rfranzke) [inline controllers/provider-openstack/charts/provider-openstack/templates/rbac.yaml]:
```suggestion
  - backupbuckets/status
```

**Comment by swapnilgm** [comment]
Done

**Comment by swapnilgm** [comment]
Done. Rebased and generated controller-registraions once again.

**Pull request merged**